feature/gnoi/system/tests/supervisor_switchover_test/supervisor_switchover_test.go

```diff
@@ -81,6 +80,13 @@ func TestSupervisorSwitchover(t *testing.T) {
 	rpStandbyBeforeSwitch, rpActiveBeforeSwitch := findStandbyRP(t, dut, supervisors)
 	t.Logf("Detected rpStandby: %v, rpActive: %v", rpStandbyBeforeSwitch, rpActiveBeforeSwitch)
 
+	switchoverReady := dut.Telemetry().Component(rpActiveBeforeSwitch).SwitchoverReady()
+	switchoverReady.Await(t, 30*time.Minute, true)
+	t.Logf("SwitchoverReady().Get(t): %v", switchoverReady.Get(t))
+	if !switchoverReady.Get(t) {
+		t.Errorf("switchoverReady.Get(t): got %v, want %v", switchoverReady.Get(t), true)
```

Comment:
Ideally I'd like to see the following to avoid duplicating constant values and typos in the future:
```go
if got, want := switchoverReady.Get(t), true; got != want {
...., got, want)
}
```
But the rest of the test doesn't seem to be following this style. So I'm ok with keeping with the prevailing style here.

@liulk What do you think about this style? I think the got/want makes it clearer and less typo-prone.

Reply:
Agreed. I will make some global changes to some of my tests in the next PR.